testing/citest/spinnaker_testing/google_scenario_support.py

```diff
@@ -108,7 +108,7 @@ def add_commandline_parameters(cls, scenario_class, builder, defaults):
     builder.add_argument(
         '--test_gce_image_name',
         default=defaults.get('TEST_GCE_IMAGE_NAME',
-                             'citest-boot-image'),
+                             'apache-base-image'),
```

Comment:
Worth capturing somewhere (even if in comments here), the gcloud commands used to create that base image? We've ended up relying on images in the past where nobody was sure exactly how they had been built.

Reply:
Makes sense, just added it to the description of this PR.  Ideally this would be a Jenkins job (or some job) to refresh the image and make it reproducible, but needed to get the tests passing so just did it manually for now.